bindings/go/Makefile.am

```diff
@@ -18,7 +18,10 @@ build:
 	$(AM_V_at)$(GO) build $(top_srcdir)/bindings/go/*.go
 
 test:
-	$(AM_V_at)LD_LIBRARY_PATH=$(UCX_SOPATH) $(GO) test $(top_srcdir)/bindings/go/*.go
+	$(AM_V_at)LD_LIBRARY_PATH=$(UCX_SOPATH) $(GO) test -v $(top_srcdir)/bindings/go/*.go
+
+bench:
+	$(AM_V_at)LD_LIBRARY_PATH=$(UCX_SOPATH) $(GO) test -v -bench=. $(top_srcdir)/bindings/go/*.go
```

Comment:
don't we need to run test/bench on compiled binary files, instead of source files?

Reply:
Tests don't include into compiled binaries.